.ci/windows_ci.ps1

```diff
@@ -109,8 +109,10 @@ function Test-kivy {
     # Tests with default environment variables.
     python -m pytest --timeout=400 --cov=kivy --cov-branch --cov-report= "$(pwd)/kivy/tests"
     # Logging tests, with KIVY_LOG_MODE=TEST.
-    $env:KIVY_LOG_MODE = 'TEST'
-    python -m pytest -m logmodetest --timeout=400 --cov=kivy --cov-append --cov-report=term --cov-branch "$(pwd)/kivy/tests"
+    $env:KIVY_LOG_MODE = 'PYTHON'
+    python -m pytest -m logmodepython --timeout=400 --cov=kivy --cov-append --cov-branch "$(pwd)/kivy/tests"
```

Comment:
Is `cov-report=` missing intentionally?

Reply:
Julian-O: Well, `cov-report=` shouldn't be there because we don't want a...
[@misl6 raises eyebrow]
[Julian-O stops, re-reads the fine-print in the manual again. Oh!]
Julian-O: Ahem, as I was saying, `cov-report=` should definitely be there because we don't want a report.